detector/detector.go

```diff
@@ -391,17 +387,23 @@ func fillCertAlerts(cvedetail *cvemodels.CveDetail) (dict models.AlertDict) {
 }
 
 // detectPkgsCvesWithOval fetches OVAL database
-func detectPkgsCvesWithOval(cnf config.GovalDictConf, r *models.ScanResult) error {
-	ovalClient, err := oval.NewOVALClient(r.Family, cnf)
+func detectPkgsCvesWithOval(cnf config.GovalDictConf, r *models.ScanResult, logOpts logging.LogOpts) error {
+	client, err := oval.NewOVALClient(r.Family, cnf, logOpts)
 	if err != nil {
 		return err
 	}
-	if ovalClient == nil {
+	defer func() {
+		if err := client.CloseDB(); err != nil {
```

Comment:
Changed Client to return Pseudo instead of nil.
https://github.com/future-architect/vuls/blob/34e7a896ee840f0629ac48924a3451520127d4d6/oval/util.go#L535-L538